parts/k8s/addons/kubernetesmasteraddons-calico-daemonset.yaml

```diff
@@ -435,155 +201,171 @@ spec:
         # marks the pod as a critical add-on, ensuring it gets
         # priority scheduling and that its resources are reserved
         # if it ever gets evicted.
-        # Deprecated in 1.10, Removed in 1.11. kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods
         scheduler.alpha.kubernetes.io/critical-pod: ''
     spec:
+      nodeSelector:
+        beta.kubernetes.io/os: linux
       hostNetwork: true
       tolerations:
-        # Allow the pod to run on the master.  This is required for
-        # the master to communicate with pods.
-        - effect: NoSchedule
-          operator: Exists
+        # Make sure calico-node gets scheduled on all nodes.
```

Comment:
Please align all array items comments with the `-`

Reply:
Fixing comment indentations by a script seems to be much harder. Let me try and come back to you later.

Reply:
I fixed indentation manually. Will update script later. @sylr Please have another look. Thanks.